src/mds/snap.h

```diff
@@ -103,11 +107,13 @@ struct sr_t {
   uint64_t change_attr = 0;             // tracks changes to snap
                                         // realm attrs.
```

Comment:
add `void print(std::ostream&)` and define in `snap.cc`. Add dump for first `seq`, `created`, `last_created`, `last_destroyed`, and flags.

Reply:
?

Reply:
PTAL https://github.com/ceph/ceph/compare/6908899e659c286ccddd84905b5474a279a5d92e..d97bd2c2c655bc5fe8b181b3d5f207f54779668d